### DISCUSSION TITLE
The size of contig.fasta and scaffolds.fasta

### DISCUSSION THREAD

**Discussion by SarahIsme** [ablab/spades#747] [Q&A]
Dear developers,

I used two paired-end reads trimmed by Trimmomatic. The forward read was 18 GB and the reverse read was 17.8 GB. The command was: `/miniconda3/bin/spades.py	--meta	-1	forward.fq.gz	-2	reverse.fq.gz	-t 16	-m	40	-k	55,77,99	-o outfile` .

However the contigs.fasta was 132MB and scaffolds.fasta was also 132MB. Then the contigs.fasta was used for binning by concoct, maxbin2 and metabat2, there were only 48, 28 and 36 bins, respectively.

Is there some parameters wrong in my command line for SPAdes?

Thanks in advance!

**Comment by asl** [comment]
Looks reasonable. Why?

**Comment by SarahIsme** [comment]
[spades.log](https://github.com/ablab/spades/files/2768449/spades.log)
[report.pdf](https://github.com/ablab/spades/files/2768452/report.pdf)
There are the .log file and quality report from QUAST. I have six 2*150bp libraries, the average size of clean forward reads and reverse reads were 19GB and 18.9GB, respectively. However, the maximum and minimum size of contigs.fasta were 150MB and 87MB, respectively. The puzzling results.

**Comment by asl** [comment]
Everything seems ok to me. Why the results are "puzzling"?

**Comment by SarahIsme** [comment]
As per our experience, such size of data should get more bins with deeper taxonomic annotations , or larger contigs.fasta. This is the first time we try SPAdes based on the literatures and other people's recommendations, so there will be such questions.

**Comment by franciscozorrilla** [comment]
Hi @SarahIsme,

I am curious, how did you chose the kmer parameters? 
Did those work better than the default kmers of 21, 33, 55?
I think the number of bins you obtained is reasonable for one sample of paired end reads when you compare with papers like Finn/Segata. But then again those numbers may be lower after you quality filter with CheckM?

Best wishes,
FZ

**Comment by SarahIsme** [comment]
Dear @franciscozorrilla ,

Sorry for the late reply.

Yes, after filtration by checkM, the number of bins will be lower.

Best regards,

Mort
